content/07.00.methods.md

```diff
@@ -208,10 +208,11 @@ This complicates the derivation of an S-MultiXcan-specific solution to compute $
 To account for this, we used a submatrix $\mathbf{R}_{\ell}$ corresponding to genes that are part of LV $\ell$ only (top 1% of genes) instead of the entire matrix $\mathbf{R}$.
 This simplification is conservative since correlations are accounted for top genes only.
 Our simulations ([Supplementary Note 1](#sm:reg:null_sim)) show that the model is approximately well-calibrated and can correct for LVs with adjacent and highly correlated genes at the top (e.g., Figure @fig:reg:nulls:qqplot:lv234).
-The model can also detect LVs associated with relevant traits (Figure @fig:lv246 and Table @tbl:sup:phenomexcan_assocs:lv246) that are replicated in a different cohort (Table @tbl:sup:emerge_assocs:lv246).
+The simulation also identified 127 LVs in which the model was not well-calibrated (e.g., Figure @fig:reg:nulls:qqplot:lv914).
+As explained before, this can be attributed to the limitations in accurately computing a gene correlation matrix and as a result, we have excluded these LVs from our main analyses.
```

Comment:
```suggestion
As this can be attributed to limitations in accurately computing a gene correlation matrix, we excluded these LVs from our main analyses.
```